project/config/lgbcni/config/core.entity_form_display.node.lgbc_information.default.yml

```diff
@@ -0,0 +1,93 @@
+uuid: 70da1f81-dbf5-46bf-97d7-5cb839892767
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.node.lgbc_information.field_news_image
+    - node.type.lgbc_information
+  module:
+    - content_moderation
+    - file
+    - path
+id: node.lgbc_information.default
+targetEntityType: node
+bundle: lgbc_information
+mode: default
+content:
+  created:
+    type: datetime_timestamp
+    weight: 10
+    region: content
+    settings: {  }
+    third_party_settings: {  }
+  field_news_image:
```

Comment:
It is good to keep the naming convention the same for Unity field_photo and for it to be an entity reference field.